qa-tests-backend/src/test/groovy/GroupsTest.groovy

```diff
@@ -118,7 +118,7 @@ class GroupsTest extends BaseSpecification {
             }
         }
         for (def role : ROLES) {
-            RoleService.deleteRole(role.getName())
+            RoleService.deleteRole(role.getName(), false)
```

Comment:
Would this be legal in groovy? If not then I think it would be better to have two separate one-arg methods than have to use that crypting `false`. :-(
```suggestion
            RoleService.deleteRole(role.getName(), alsoDeletePermissionSet=false)
```